src/catalog/src/stash.rs

```diff
@@ -253,6 +253,10 @@ impl OpenableDurableCatalogState<Connection> for OpenableConnection {
 
         Ok(deployment_generation)
     }
+
+    async fn expire(self) {
+        // Nothing to release in the stash.
```

Comment:
uncovered

Reply:
Is it possible that it's marked as uncovered because it's empty? I'm pretty sure that it's covered in tests. Either way though I'm not too worried about having coverage since it's an empty method.